web_console/odc-about-web-terminal.adoc

```diff
@@ -13,7 +13,7 @@ You can launch an embedded command line terminal instance in the web console. Yo
 Cluster administrators can access the web terminal in {product-title} 4.7 and later.
 ====
 
-This terminal instance is preinstalled with common CLI tools for interacting with the cluster, such as `oc`, `kubectl`,`odo`, `kn`, `tkn`, `helm`, `kubens`, `subctl`, and `kubectx`. It also has the context of the project you are working on and automatically logs you in using your credentials.
```

Comment:
Same comment as above.

Reply:
Thanks for your feedback. Revoking the change here.